src/app/organisations/deleteUserOrganisation.js

```diff
@@ -1,11 +1,31 @@
-const { deleteUserOrganisation } = require('./data/organisationsStorage');
+const { deleteUserOrganisation, getServiceAndSubServiceReqForOrgs, updateUserServSubServRequest } = require('./data/organisationsStorage');
 
 const deleteOrg = async (req, res) => {
-  const organisationId = req.params.id;
-  const userId = req.params.uid;
+  const correlationId = req.get('x-correlation-id');
+  const { id: orgId, uid: userId } = req.params;
```

Comment:
it can go like this const { orgId, userId } = req.params;
id, uid are not being used